cleanlab/datalab/internal/issue_manager/duplicate.py

```diff
@@ -233,3 +239,40 @@ def _set_threshold(
             )
             threshold = 0
         return threshold
+
+
+def _compute_scores_with_exp_transform(nn_distances: np.ndarray, temperature: float) -> np.ndarray:
+    r"""Compute near-duplicate scores from nearest neighbor distances.
+
+    This is a non-linear transformation of the nearest neighbor distances that
+    maps distances to scores in the range [0, 1].
+
+    Note
+    ----
+
+    This transformation is given by the following formula:
+
+    .. math::
+
+        \text{score}(d, t) = 1 - e^{-dt}
+
+    where :math:`d` is the nearest neighbor distance and :math:`t > 0` is a temperature parameter.
+
+    Parameters
+    ----------
+    nn_distances :
+        The nearest neighbor distances for each example.
+
+    Returns
+    -------
+    scores :
+        The near-duplicate scores for each example. The scores are in the range [0, 1].
+        A lower score indicates that an example is more likely to be a near-duplicate than
+        an example with a higher score.
+        A score of 0 indicates that an example has an exact duplicate.
+    """
```

Comment:
Most of this docstring was written for the docs page, but it won't show up as it's an internal/private function.

It's fine to remove this docstring to reduce clutter.